cmake/CompilerOptions.cmake

```diff
@@ -4,6 +4,8 @@ set(CXXOPT_DEBUG "-g")
 set(CXXOPT_PTHREAD "-pthread")
 set(CXXOPT_CXX11 "-std=c++11")
 set(CXXOPT_WALL "-Wall")
+set(BSCOPT_EXTRAE "-O -g -I $(EXTRAE_HOME)/include")
+set(BSCLINK_EXTRAE "-Wl,-rpath -Wl,/apps/CEPBATOOLS/extrae/latest/impi5_mt+libgomp4.9/64/lib -L/apps/CEPBATOOLS/extrae/latest/impi5_mt+libgomp4.9/64/lib")
```

Comment:
The way to do this properly would be to write a CMake module, something like `extrae.find`, to automate this process.